### DISCUSSION TITLE
When sending assets, the gas fee automatically obtained by the LND node is too small, resulting in miners never packing. What about this?

### DISCUSSION THREAD

**Discussion by fanshome1001** [lightninglabs/taproot-assets#1012] [Q&A]
When sending assets, the gas fee automatically obtained by the LND node is too small, resulting in miners never packing. What about this?
txid : ce3c997a2d055fe1e6e342a65f01c12052a80b5047acf6636e05c05227c268d6
![tx](https://github.com/lightninglabs/taproot-assets/assets/146626011/5d364606-cedf-45b9-b84e-8b520649e20c)
[https://mempool.space/zh/address/bc1pzy57gk7h3zx7tn0sxmxkrnws4f2ffv70ucr7j0vjcal3u6qxqr6qpc2spe](url)

**Comment by guggero** [comment]
What chain backend are you using for `lnd`? Bitcoin Core or `btcd` or Neutrino?
You can avoid this if you include a fee rate in the `tapcli assets mint finalize` call, for example `tapcli assets mint finalize --fee_rate=xxx` where `xxx` is in satoshi per kWU (`satoshi_per_vbyte * 250`), so for example if you want to use 9 sat/vByte, you'd use 2250.
In the upcoming `v0.4.0-alpha` release this will be easier and the new `--sat_per_vbyte` flag directly.

**Reply by fanshome1001** [reply]
I used Neutrino。
What should I do about this deal? Is it possible to withdraw the deal?

**Reply by guggero** [reply]
What fee URL did you configure? Neutrino doesn't have its own estimation so it relies on an URL to tell you the current fee. It sounds like it gave you a bad fee estimation.
You can accelerate the transaction by attaching more fees to it in a so called "child" transaction. You basically spend the change output of the transaction shown above with a transaction that pays more in fees.
You can use `lncli wallet bumpfee ce3c997a2d055fe1e6e342a65f01c12052a80b5047acf6636e05c05227c268d6:2 --sat_per_vbyte XXX`, (choose XXX by looking at mempool.space and choosing a high enough value) which when successful will give you a response like `{}`. But you can then watch the transaction above on mempool.space and see that it'll eventually have its third output being spent and hopefully confirmed.

**Reply by jharveyb** [reply]
For transfers, the flag to set a manual feerate is `--sat_per_vbyte`.

**Comment by fanshome1001** [comment]
When is version v0.4.0-alpha expected to come online?

**Reply by guggero** [reply]
Soon. A first RC is already available, though I wouldn't recommend updating any production or mainnet infrastructure to it yet.

**Reply by jharveyb** [reply]
v0.4.1 is now released.
